modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -2072,6 +2072,63 @@ CV_EXPORTS_W void HoughLinesP( InputArray image, OutputArray lines,
                                double rho, double theta, int threshold,
                                double minLineLength = 0, double maxLineGap = 0 );
 
+/** @brief Finds lines in a set of points using the standard Hough transform.
+
+The function finds lines in a set of points using a modification of the Hough transform.
+
+Example: :
+@code
+    #include <opencv2/imgproc.hpp>
+    #include <opencv2/highgui.hpp>
+    #include <math.h>
```

Comment:
Code in commends is not tested at all, and become dead/wrong very quickly. So we don't allow this.
Use snippets instead to check code compilation at least. Check for [this patch](https://github.com/opencv/opencv/pull/10512/files#diff-b6033f44ae355b1c40481eca8b593a89R200), snippets [location](https://github.com/opencv/opencv/tree/master/samples/cpp/tutorial_code/snippets).

Reply:
I’m so sorry, I could not maintain the sample code at all.
I fixed sample code. Thank you for the feedback.

Reply:
I mean that sample code should not be stored in comments. See this commit how to store function documentation samples: https://github.com/opencv/opencv/commit/11fa0094ffee2437ade42f99847d6c0485e0a360

Reply:
I understood what you mean. I'm so sorry again. I work on it.
Can I add a new file for sample?

Reply:
Yes, "imgproc_HoughLinesPointSet.cpp" should be fine.

Reply:
Thank you for teaching me a lot.
1. Deleted sample code in comment.
2. Added snippets for HoughLinesPointSet.